aepsych/models/pairwise_probit.py

```diff
@@ -99,6 +119,13 @@ def fit(
         optimizer_kwargs: Optional[Dict[str, Any]] = None,
         **kwargs,
     ) -> None:
+        """Fit the model to the training data.
+
+        Args:
+            train_x (torch.Tensor): Trainin x points.
+            train_y (torch.Tensor): Training y points.
+            optimizer_kwargs (Optional[Dict[str, Any]], optional): Keyword arguments to pass to the optimizer. Defaults to None.
+        """
```

Comment:
Are the **kwargs actually useful? If not, it should be removed from the function signature.

Reply:
This is a functional change, and I'm not sure if it is useful or not. I'll need to test it and potentially implement this adjustment in a separate PR.